### ISSUE TITLE
Support clicks/dragging inside the circle

### ISSUE THREAD

**Issue by tbrannt** [soundar24/roundSlider#55]
I switched from https://github.com/princejwesley/circular-slider to this slider because the circular slider didn't support changing the the sliders value when the mouse went out of the circle. I was happy to see that this intuitive behavior is supported by the roundSlider.

Unfortunately in contrast to circular-slider the roundSlider from this site does not support dragging with the mouse inside the circle. Especially for use on mobile devices you're more or less forced to choose a broad slider bar. I think it would be a clear enhancement without any drawbacks to also support clicking and dragging inside the roundSlider as also https://github.com/princejwesley/circular-slider supports.

**Comment by soundar24** [comment]
Actually I considered this at the initial time itself, you can check the below report regarding the requirement:
https://github.com/soundar24/roundSlider/issues/4

At that time due to other things concentration i didn't considered this, but i already provided the workaround solution for this requirement:

## http://jsfiddle.net/soundar24/utqgufx0/295/

Please check the above demo and confirm me whether this is your required behavior, if yes then i will include in the source soon (maybe within 1 or 2 weeks).

**Comment by tbrannt** [comment]
That's exactly the behavior I was thinking about! Thanks very much for the fast reply. Having this in the next version would be awesome.

**Comment by soundar24** [comment]
Yes, I will consider this and it will be available in the next release. Thanks :)

**Comment by tbrannt** [comment]
ölaksd öfk<br>k  aöklsdf 
aklsödfa
dasd<a href="#">link</a> kflö 
asdfas

```
for(var i = 0; i < lines.length; i++) {
	console.log(lines[i]);
	fox.mdArea[0].setSelectionRange(startPos, startPos + lines[i].length);
	fox.surroundOrDesurroundSelOrWordWithDifferent('> ', '  ');
				
	startPos+=(lines[i].length + insertedLen + skippedLen);
}
```
jkljlklklk
